src/lib/ui/LoadingToast/LoadingToast.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames'
+import { useEffect, useState } from 'react'
+
+import { CheckBlackIcon } from 'lib/icons/CheckBlack.js'
+import { Spinner } from 'lib/ui/Spinner/Spinner.js'
+
+interface LoadingToastProps {
+  isLoading: boolean
+  isDoneLoading?: boolean
+  top?: number
+  left?: number
+}
+
+export function LoadingToast({
+  isLoading,
+  isDoneLoading = false,
+  top,
+  left,
+}: LoadingToastProps): JSX.Element {
+  const [shouldRender, setShouldRender] = useState(true)
```

Comment:
Render suggests a react rendering, hidden suggests a style, if that makes sense. 

```suggestion
  const [hidden, setHidden] = useState(false)
```